prov/util/src/util_shm.c

```diff
@@ -166,8 +195,13 @@ int smr_create(const struct fi_provider *prov, struct smr_map *map,
 
 	fd = shm_open(attr->name, O_RDWR | O_CREAT | O_EXCL, S_IRUSR | S_IWUSR);
 	if (fd < 0) {
-		FI_WARN(prov, FI_LOG_EP_CTRL, "shm_open error\n");
-		return -errno;
+		ret = smr_retry_map(attr->name, &fd);
+		if (ret) {
+			FI_WARN(prov, FI_LOG_EP_CTRL, "shm_open error\n");
```

Comment:
While you're here can you add a `strerror(ret)` and the file name to this warn?

Reply:
Will do